lib/constants.ts

```diff
@@ -69,3 +69,18 @@ export const PLATFORM_VERSION_ERROR_TYPES = {
   PLATFORM_VERSION_SPECIFIED_DOES_NOT_EXIST:
     'PlatformVersionErrorType.PLATFORM_VERSION_SPECIFIED_DOES_NOT_EXIST',
 } as const;
+
+export const IR_COMPONENT_TYPES = {
```

Comment:
Might be nice to get these from the parsing lib eventually. But keeping them here is fine for now 👍

Reply:
Yeah, we can get these from the parsing lib eventually.  We have a list of the valid component keys, and a method to translate them to the internal values which is what is shown in the IR.